src/dali_backend.h

```diff
@@ -144,42 +48,42 @@ std::vector<IODescr<true>> GenerateInputs(TRITONBACKEND_Request* request) {
  * processing.
  * @return A descriptor that wraps Triton output
  */
-std::vector<IODescr<false>> AllocateOutputs(
-    TRITONBACKEND_Request* request, TRITONBACKEND_Response* response,
-    const std::vector<shape_and_type_t>& shapes_and_types,
-    const std::unordered_map<std::string, int>& output_order) {
+std::vector<ODescr> AllocateOutputs(TRITONBACKEND_Request* request,
```

Comment:
Why not `TritonRequest` here?

Reply:
I will move the whole response and outputs logic to the C++ API in the next PR

Reply:
If anything, it should be the view (non-owning).